RELEASE.md

```diff
@@ -0,0 +1,3 @@
+Release type: patch
+
+We change the logging format at `process_errors` for `GraphQLError` errors.
```

Comment:
``````suggestion
This release improves the default logging format for errors to include more information about the errors. For example it will show were an error was originated in a request:

```
GraphQL request:2:5
1 | query {
2 |     example
  |     ^
3 | }
```
``````